hedera-node/log4j2.xml

```diff
@@ -198,9 +198,9 @@
     </Logger>
 
     <!-- Send transaction state logs to their own appender   -->
-    <AsyncLogger name="com.hedera.node.app.state.logging.TransactionStateLogger" level="trace" additivity="false">
+    <Logger name="com.hedera.node.app.state.logging.TransactionStateLogger" level="trace" additivity="false">
```

Comment:
I thought it was supposed to be asyncLogger. why do we change it to Logger?

Reply:
Because It is not supposed to be async yet, as it pulls in a dependency we don't want to bring in yet.
Jasper asked for this change via a slack thread.